src/privatesend/privatesend-server.cpp

```diff
@@ -628,48 +538,128 @@ bool CPrivateSendServer::IsInputScriptSigValid(const CTxIn& txin)
 }
 
 //
-// Add a clients transaction to the pool
+// Add a client's transaction to the pool
```

Comment:
it's not really adding a transaction, more like adding inputs, since you're already changing this line, might as well change that too

Reply:
How about
```suggestion
// Add a client's transaction parts to the pool
```

Reply:
Sounds a little awkward imo, 
```suggestion
// Add a client's inputs to the pool
```
but I don't have a strong opinion. At this point their sigs aren't added or anything, it really is just the input

Reply:
it's inputs and outputs here

Reply:
:+1: